helpdesk_stock/__manifest__.py

```diff
@@ -11,10 +11,12 @@
     'depends': [
         'helpdesk_timesheet',
         'stock_request_picking_type',
+        'stock_request'
```

Comment:
@osi-scampbell No need to add it. It is already a dependency of `stock_request_picking_type`